### PR TITLE
add challenges

### PR DESCRIPTION
**Pull request by Lohikeitto**

### CODE REVIEW

File: staff/blas-toth/challenges/calculadora/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Calulator App
+
+## Intro
+
+blah blah blah...
```
Comment (betoDavila86) [inline staff/blas-toth/challenges/calculadora/README.md]: Missing description :)

File: staff/blas-toth/challenges/calculadora/main.js [hunk 2]
```diff
@@ -0,0 +1,262 @@
+var memory = ''
+var auxiliar
```
Comment (betoDavila86) [inline staff/blas-toth/challenges/calculadora/main.js]: Don't forget the semicolons ;

**Review by betoDavila86 — CHANGES_REQUESTED** [review]

File: staff/blas-toth/challenges/calculadora/main.js [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+button7.onclick = number7;
+
+function number7() {
+
+        var display = document.querySelector(".display");
+
+        memory += 7
+
+        display.innerText = memory;
+
```
Comment (Crisappletree) [inline staff/blas-toth/challenges/calculadora/main.js]: remove empty lines

**Review by simonbesteiro — APPROVED** [review]
review done

File: staff/blas-toth/challenges/calculadora/README.md (hunk 1 above)
Comment (Lohikeitto) [reply staff/blas-toth/challenges/calculadora/README.md]: Thank you, I have added the description.

File: staff/blas-toth/challenges/calculadora/calculate.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function calculate(number1, operation, number2) {
+    // TODO implement the four operations + - / *
+
+    if (operation === '+') {
+        result = Number(auxiliar) + Number(memory)
```
Comment (manuelbarzi) [inline staff/blas-toth/challenges/calculadora/calculate.js]: add semicolon in all statements where you assign values (=)

File: staff/blas-toth/challenges/calculadora/main.js (hunk 2 above)
Comment (Lohikeitto) [reply staff/blas-toth/challenges/calculadora/main.js]: Consider it done ;

File: staff/blas-toth/challenges/calculadora/calculate.js (hunk 4 above)
Comment (Lohikeitto) [reply staff/blas-toth/challenges/calculadora/calculate.js]: Thank you, done!

File: staff/blas-toth/challenges/calculadora/main.js (hunk 3 above)
Comment (Lohikeitto) [reply staff/blas-toth/challenges/calculadora/main.js]: done

**Review by simonbesteiro — APPROVED** [review]
review done

**Pull request merged**